tests/shared/src/test/scala/scala/meta/tests/parsers/ParseSuite.scala

```diff
@@ -51,12 +51,12 @@ class ParseSuite extends FunSuite with CommonTrees {
 }
 
 object MoreHelpers {
-  def requireNonEmptyOrigin(tree: Tree): tree.type = {
+  def requireNonEmptyOrigin(tree: Tree)(implicit dialect: Dialect): tree.type = {
     val missingOrigin = tree.collect {
```

Comment:
So this actually relies on the `children` method, which was not working for `binaryCompat` fields previosuly and now that it started working it turns out that origin is not set for a number of cases. They should be fixed now.

I also added dialect and `.map(_.structure)` otherwise an exception was thrown while trying to print the trees.

Reply:
This also explains why I was having problems getting a proper formatting for `inline if` :sweat_smile: